tools/accuracy_checker/configs/resnet-50-tf.yml

```diff
@@ -2,11 +2,7 @@ models:
   - name: resnet-50-tf
     launchers:
       - framework: tf
-        model: public/resnet-50-tf/resnet_v1-50.pb
-        adapter: classification
-        inputs:
-          - name: map/TensorArrayStack/TensorArrayGatherV3
-            type: INPUT
+        # TODO
```

Comment:
@eaidova Is there a way to make Accuracy Checker work with the original SavedModel?

Reply:
not sure about this model specific parameters, but you can try to use
`saved_model_dir` instead of `model`

Reply:
it is experimental feature in AC, so I am checking it.

Reply:
not sure that it is possible, saved_model contains image loading nodes if i right understand, AC can not remove it.
the same model without jpg suffix loaded, but fails due to uninitilized variables...

Reply:
Would it maybe be possible to replicate the functionality of [`strip_unused_lib`](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/tools/strip_unused_lib.py) to remove the unneeded nodes?

Reply:
Actually, if we could use the non-JPEG model, that would be even better. I checked just now and MO can convert it without issues, so clearly the model itself is fine. Maybe AC just doesn't load it correctly?